### DISCUSSION TITLE
Tree font?

### DISCUSSION THREAD

**Discussion by stevesobol** [lukasbach/react-complex-tree#274] [Q&A] [answered]
Hello,

Is it possible to get the tree to display using any font other than `sans-serif`?

I have a specific Google font I want to use instead.

Thanks

**Comment by lukasbach** [comment, marked as answer]
Rendering is completely up to you. If you are using default renderers and default styles, you can look at how the default styles look like [here](https://github.com/lukasbach/react-complex-tree/blob/main/packages/core/src/style.css). You can update the font family in the `.rct-tree-root` class. If you are using custom renderers, the classes to restyle are dependent on your code.
